crates/rattler/src/solver/libsolv/repo.rs

```diff
@@ -173,21 +173,22 @@ impl RepoRef {
                 )
             };
 
-            // Location (filename (fn) and subdir)
-            let filename = record
-                .filename
-                .as_deref()
-                .unwrap_or_else(|| filename.as_ref());
-            let subdir = record.subdir.as_str();
-            unsafe {
-                ffi::repodata_set_location(
-                    data,
-                    solvable_id.into(),
-                    0,
-                    CString::new(subdir)?.as_ptr(),
-                    CString::new(filename)?.as_ptr(),
-                );
-            }
+            // TODO: The filename is not present in a PackageRecord
+            // // Location (filename (fn) and subdir)
+            // let filename = record
+            //     .filename
+            //     .as_deref()
+            //     .unwrap_or_else(|| filename.as_ref());
+            // let subdir = record.subdir.as_str();
+            // unsafe {
+            //     ffi::repodata_set_location(
+            //         data,
+            //         solvable_id.into(),
+            //         0,
+            //         CString::new(subdir)?.as_ptr(),
+            //         CString::new(filename)?.as_ptr(),
+            //     );
+            // }
```

Comment:
We should probably get rid of this commented-out code

Reply:
I can just remove it. I assumed that in your branch this will work slightly different anyway?

Reply:
Or actually. I think the `add_repodata` function should take `RepoDataRecord` as its argument instead of the RepoData directly.

Reply:
Hmm I don't understand this code enough yet to know what we need here, I just thought you had forgotten to remove the comments 😅

Reply:
No I think we do need this, (not entirely sure), so I left it for future reference.